hack/test-unit.sh

```diff
@@ -26,3 +26,20 @@ do
     os::log::info "Testing ${PACKAGE}"
     go test $PACKAGE
 done
+
+OCP_PACKAGES_TO_TEST=(
```

Comment:
I wonder if we can deduplicate somehow.  IE,

```
PACKAGES_TO_TEST=(/* most go here */)
PACKAGES_TO_TEST_FOR_OKD=(/* just one? */)
PACKAGES_TO_TEST_FOR_OCP=(/* just one? */)
// then always run the main set, but conditionally run `_FOR_OKD` & `_FOR_OCP`.  
```
Would this just be a flag to check to run one more loop?

Reply:
This is probably not a big deal. Resolving, if we ever had more complexity we can worry about it then.